### PR TITLE
OSDOCS#9413: OLM 1.0 supports AllNamespace install and no webhooks

### PR DESCRIPTION
**Pull request by michaelryanpeter**
* Updating the "Finding an Operator to install docs" to clarify that OLM 1.0 only supports the `AllNamespaces` install mode and does not support webhooks
* Update common queries to include a search for versions in a channel and for extensions supported by OLM 1.0

Version(s): 4.15

Issue: [OSDOCS-9413](https://issues.redhat.com//browse/OSDOCS-9413)

Link to docs preview:
* [Finding Operators to install from a catalog (Step 3 now has an admonition and substeps)](https://70584--ocpdocs-pr.netlify.app/openshift-enterprise/latest/operators/olm_v1/olmv1-installing-an-operator-from-a-catalog#olmv1-finding-operators-to-install_olmv1-installing-operator)
* [Common queries > Table 2. Common channel queries > Versions in a channel and Packages that support AllNamespaces mode and don't use webhooks](https://70584--ocpdocs-pr.netlify.app/openshift-enterprise/latest/operators/olm_v1/olmv1-installing-an-operator-from-a-catalog#olmv1-catalog-queries_olmv1-installing-operator)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/olmv1-finding-operators-to-install.adoc [hunk 1]
```diff
@@ -111,10 +112,62 @@ $ jq -s '.[] | select( .schema == "olm.package") | \
...
++
+[source,terminal]
+----
+$ opm render --migrate <catalog_reference> | \
+  jq 'select(.schema == "olm.bundle" and (.properties[] | \
+  select(.type == "olm.csv.metadata" \
+  and (.value.installModes[] | \
+  select(.type == "AllNamespaces" and .supported == true) \
+  != null)) != null)).name'
+----
```
Comment (michaelryanpeter) [inline modules/olmv1-finding-operators-to-install.adoc:133]: @joelanford Can we modify this query to also search for bundles that do not include webhooks?

**Review by everettraven — APPROVED** [review]
LGTM

**Comment by michaelryanpeter** [comment]
@jianzhangbjz PTAL :)

**Comment by Xia-Zhao-rh** [comment]
LGTM
/label qe-approved

**Comment by nalhadef** [comment]
Thanks, Michael! Looks good!

**Comment by michaelryanpeter** [comment]
Thanks, Neal!

**Pull request merged**